metals/src/main/scala/scala/meta/internal/metals/debug/MetalsDebugAdapter.scala

```diff
@@ -0,0 +1,54 @@
+package scala.meta.internal.metals.debug
+
+import java.util.concurrent.CompletableFuture
+import org.eclipse.lsp4j.debug.Capabilities
+import org.eclipse.lsp4j.debug.InitializeRequestArguments
+import org.eclipse.lsp4j.debug.services.IDebugProtocolClient
+import org.eclipse.lsp4j.jsonrpc.services.JsonRequest
+import scala.concurrent.ExecutionContext
+import scala.meta.internal.debug.DebugAdapter
+import scala.meta.internal.debug.JvmDebugAdapter
+import scala.meta.internal.metals.BuildTargets
+import scala.meta.internal.metals.CompilationScheduler
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.debug.{protocol => metals}
+
+/**
+ * Metals receive requests without the full knowledge - e.g. without classpath.
```

Comment:
When writing docstrings for classes like this, try to explain the motivation for the existence of this class instead of implementation specifics.

> MetalsDebugAdapter is similar to MetalsLanguageServer except it implements the Debug Adapter Protocol instead of Language Server Protocol.